### PR TITLE
[ja] Translating core-libraries/text.

### PR DESCRIPTION
**Pull request by waterada**
Please don't yet merge this. Now translating.

### CODE REVIEW

**Comment by chinpei215** [comment]
@waterada Thank you. Could you please remove original text from your translation, as it makes things complex.

File: ja/core-libraries/text.rst [hunk 1]
```diff
@@ -5,21 +5,383 @@ Text
...
+
+    echo Text::highlight(
+        $lastSentence,
+        'using',
+        ['format' => '<span class="highlight">\1</span>']
+    );
+
+出力::
+
+    $options['format'] で指定された文字列か、デフォルトの文字列を<span class="highlight">使って</span>
```
Comment (chinpei215) [inline ja/core-libraries/text.rst]: This output is incorrect, as 'using' is passed in the example above.

File: ja/core-libraries/text.rst [hunk 2]
```diff
@@ -5,21 +5,383 @@ Text
...
+    handy for search results. The query...
+
+
+配列を文章的なものに変換する
+====================================
+
+.. php:method:: toList(array $list, $and='and', $separator=', ')
+
+最後の２要素が 'and' で繋がっている、カンマ区切りのリストを生成します::
+
```
Comment (chinpei215) [inline ja/core-libraries/text.rst]: You removed the following line from this example accidentally:
> $colors = ['red', 'orange', 'yellow', 'green', 'blue', 'indigo', 'violet'];

File: ja/core-libraries/text.rst [hunk 3]
```diff
@@ -5,21 +5,383 @@ Text
...
+    // Called as TextHelper
```
Comment (chinpei215) [inline ja/core-libraries/text.rst]: Untranslated.

File: ja/core-libraries/text.rst [hunk 4]
```diff
@@ -5,21 +5,383 @@ Text
...
+    echo $this->Text->toList($colors);
+
+    // Called as Text
```
Comment (chinpei215) [inline ja/core-libraries/text.rst]: Same as above.

File: ja/core-libraries/text.rst [hunk 5]
```diff
@@ -5,21 +5,383 @@ Text
...
+    // foo-bar.tar.gz
+    Text::slug('foo bar.tar.gz', ['preserve' => '.']);
+
+
+UUIDの生成
+================
+
+.. php:staticmethod:: uuid()
+
+UUIDメソッドは :rfc:`4122` 準拠のユニークな識別子を生成するのに使います::
```
Comment (waterada) [inline ja/core-libraries/text.rst]: Thanks. I will fix it.

File: ja/core-libraries/text.rst [hunk 6]
```diff
@@ -5,21 +5,383 @@ Text
...
+        'My name is :name and I am :age years old.',
+        ['name' => 'Bob', 'age' => '65']
+    );
+    // これを返す: "My name is Bob and I am 65 years old."
+
+.. php:staticmethod:: cleanInsert($string, $options = [])
+
+``$options`` 内の 'clean' キーに従って、 ``Text::insert`` でフォーマットされた文字列を掃除します。
+デフォルトで method に使われるのは text ですが html も使えます。
+この機能の目的は、``Text::insert`` で置き換えられなかった、プレイホルダ周辺のすべての空白と不要なマークアップを置き換えることにあります。
```
Comment (waterada) [inline ja/core-libraries/text.rst]: Thanks. I will fix it.

**Comment by chinpei215** [comment]
@waterada Thank you. Nice work.

**Pull request merged**